src/engine-actions/create-channels.js

```diff
@@ -0,0 +1,186 @@
+const { CHANNEL_ROUNDING } = require('../constants')
+
+/**
+ * Determine the amount to fund channels with by taking into account
+ * uneconomic channels (that are too small to be worth it.)
+ * @param   {Engine} engine
+ * @param   {string} fundingAmount - int64 string of amount desired to open
+ * @param   {number} roundBehavior - Behavior to use when encoutnering a channel that is too small.
+ * @returns {Big}                    Modified amount to open in channels
+ * @private
+ */
+function getAmountForChannels (engine, fundingAmount, roundBehavior) {
+  const {
+    maxChannelBalance,
+    minChannelBalance,
+    quantumsPerCommon,
+    symbol,
+    logger
+  } = engine
+
+  let amount = Big(fundingAmount)
+
+  // our last channel will be our only channel not at maxChannelBalance, so we need to handle it separately
+  const lastChannelBalance = amount.mod(maxChannelBalance)
+
+  if (lastChannelBalance.gt(0) && lastChannelBalance.lt(minChannelBalance)) {
+    logger.debug('Last channel would result in an uneconomic channel balance.', { fundingAmount, lastChannelBalance })
+    const lowFundingAmount = amount.minus(lastChannelBalance)
+    const highFundingAmount = lowFundingAmount.plus(minChannelBalance)
+
+    if (roundBehavior === CHANNEL_ROUNDING.ERROR) {
+      throw new Error(`Funding amount would result in an uneconomic channel balance. ` +
+        `Try either ${lowFundingAmount.div(quantumsPerCommon).toString()} ${symbol} ` +
+        `or ${highFundingAmount.div(quantumsPerCommon).toString()} ${symbol}.`)
+    }
+
+    if (roundBehavior === CHANNEL_ROUNDING.DOWN) {
+      logger.debug('Rounding down to avoid uneconomic channel balance.', { fundingAmount, lowFundingAmount: lowFundingAmount.toString() })
+      if (lowFundingAmount.lte(0)) {
+        throw new Error(`Funding amount ${amount.div(quantumsPerCommon).toString()} ${symbol} ` +
+          `too small for minimum channel balance of ${Big(minChannelBalance).div(quantumsPerCommon)} ${symbol}.`)
+      }
+      return lowFundingAmount
+    }
+
+    if (roundBehavior === CHANNEL_ROUNDING.UP) {
+      logger.debug('Rounding up to avoid uneconomic channel balance.', { fundingAmount, highFundingAmount: highFundingAmount.toString() })
+      return highFundingAmount
+    }
+
+    throw new Error(`Invalid round behavior: ${roundBehavior}. Must be one of ${Object.values(CHANNEL_ROUNDING)}.`)
+  }
+
+  return amount
+}
+
+/**
+ * Check that the balance in the engine can cover the desired funding amount
+ * @param   {Engine} engine
+ * @param   {string} fundingAmount - Int64 string of desired funding amount
+ * @returns {void}
+ * @throws {Erro} If Balance is insufficient to cover amount and fees
+ */
+async function assertBalance (engine, fundingAmount) {
+  const {
+    maxChannelBalance,
+    quantumsPerCommon,
+    feeEstimate,
+    symbol,
+    logger
+  } = engine
+
+  const amount = Big(fundingAmount)
+
+  // round up to find the number of channels to open
```

Comment:
Is there a reason you dropped the link to the `round` docs? Might be useful for folks looking at the code for the first time.

That said, it's also obvious this is a method on `big.js`.